src/main/resources/static/api.yaml

```diff
@@ -699,19 +699,6 @@ components:
         system:
           type: string
           format: uri
-    HITypeRepresentation:
```

Comment:
What is the reason behind it?

Reply:
HiTypeEnum is kind of an id/identifier (for machine), and description is  for human, that was the reason behind having it in the first place.

Reply:
I  have removed the HITypeRepresentation. The HITypes are known "constants"/"valye-sets", and will be documented in CR or by other authority. All components if they need the textual description, they can keep a known list in their own systems.